lib/Mojolicious/Plugin/PODRenderer.pm

```diff
@@ -96,7 +96,7 @@ sub _pod_to_html {
   my $parser = Pod::Simple::XHTML->new;
   $parser->perldoc_url_prefix('https://metacpan.org/pod/');
   $parser->$_('') for qw(html_header html_footer);
-  $parser->strip_verbatim_indent(sub { my ($i) = $_[0][0] =~ /^(\s+)/; $i });
+  $parser->strip_verbatim_indent(sub { (sort map /^(\s+)/, @{$_[0]})[0] });
```

Comment:
that was actually my complaint here: http://irclog.perlgeek.de/mojo/2015-05-09#i_10574016